AirCasting/CreateSessionViews/WifiPopupView.swift

```diff
@@ -29,9 +29,13 @@ struct WifiPopupView: View {
                 connectToDifferentWifi
             }
             VStack(spacing: 10) {
-                Button("Connect") {
-                    presentationMode.wrappedValue.dismiss()
-                } .buttonStyle(BlueButtonStyle())
+                if wifiSSID != "" && wifiPassword != "" {
+                    confirmationButton
+                } else {
+                    confirmationButton
+                    .accentColor(.gray)
+                    .disabled(true)
```

Comment:
Considering what @mateuszmackowiak wrote, there's a possibility not to have a wifi password. So I guess we shouldn't disable connect button when the password is empty. :<

Reply:
But you can leave some of the logic, and check `if wifiSSID.isEmpty` 🤔